src/main/java/org/minions/devfund/aldobalderrama/WhackAMole.java

```diff
@@ -0,0 +1,131 @@
+package org.minions.devfund.aldobalderrama;
+
+/**
+ * This class handle the WhackAMole game.
+ */
+class WhackAMole {
+    private static final int SCORE_POINT = 10;
+    private int score;
+    private int molesLeft;
+    private int attemptsLeft;
+    private int gridDimension;
+    private int moles;
+    private char[][] moleGrid;
+
+    /**
+     * Constructor a new instance.
+     *
+     * @param numAttempts   the number Attempts.
+     * @param gridDimension the grid Dimension.
+     */
+    WhackAMole(final int numAttempts, final int gridDimension) {
+        score = 0;
+        moles = gridDimension;
+        molesLeft = gridDimension;
+        attemptsLeft = numAttempts;
+        this.gridDimension = gridDimension;
+        moleGrid = new char[gridDimension][gridDimension];
+        fillGrid();
+    }
+
+    /**
+     * Fills the grid.
+     */
+    private void fillGrid() {
+        for (int column = 0; column < gridDimension; column++) {
+            for (int row = 0; row < gridDimension; row++) {
+                moleGrid[column][row] = '*';
+            }
+        }
+    }
+
+    /**
+     * Prints the grid to user.
+     */
+    void printGridToUser() {
+        for (int column = 0; column < gridDimension; column++) {
+            for (int row = 0; row < gridDimension; row++) {
+                if (moleGrid[column][row] == 'M') {
+                    System.out.print("*");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)